### ISSUE TITLE
paper-button and paper-material styles don't coordinate.

### ISSUE THREAD

**Issue by notwaldorf** [PolymerElements/paper-button#9]
I have:

```
<style is="custom-style">
.button {
    padding-right: 10px;
    background-color: #4285f4;
    color: #fff;
    height: 30px;
  }
</style>
```
and `<paper-button class="button" raised>eeeek</paper-button>` just dumped in the body, and this ends up looking like this:

![screen shot 2015-05-11 at 12 23 04 pm](https://cloud.githubusercontent.com/assets/1369170/7573192/b0588f92-f7d8-11e4-9cbf-438bf10c5ed2.png)

**Comment by Rich17** [comment]
I have the same issue. Setting the paper-material height helps, but the vertical alignment of the button text is still off:

    .button::shadow paper-material{
        height:8px;
    }

**Comment by Rich17** [comment]
Found a work-around: the problem is resolved when you manipulate the size of the button by adjusting the font-size only, which in turn resizes the button

**Comment by dgtlife** [comment]
When I compared the old `paper-button` (0.5.5) to the new `paper-button` (1.0.1) I noticed three key things:
* a `paper-material` element has replaced the `paper-shadow` element which was used in 0.5.6 to provide the _raised_ effect
* the `paper-material` element has been given the _wrong_ CSS and does not fill the button as the `paper-shadow` did in 0.5.5
* there is no longer a `div.button-content` to which padding was applied to fill out the button to a reasonable size surrounding the label and any icon when included.

Once the CSS that was applied to `paper-shadow` (via the `fit` class) is applied to `paper-material`, as in:
```
position: absolute;
top: 0;
bottom: 0;
left: 0;
right: 0;
```
and the padding that was applied to `div.button-content` is applied to `paper-button`, as in:
```
padding: 0.7em 0.57em;
```
we achieve equivalent appearance between the 0.5 paper-button and the 1.0 paper-button.

I believe this remedy can and should be applied at the component level.
